### PR TITLE
cmdline parameters and non-signed test repo for debian

### PR DESCRIPTION
**Pull request by mdubbyap**
provide command line parameters at top level and pass those to install.sh
use non-signed test repo for debian when --test is passed 
updated for new collectd sfx8

### CODE REVIEW

**Comment by mdubbyap** [comment]
@tedoc2000  wouldn't mind your eyeballs on this as well

**Comment by mdubbyap** [comment]
@udaysagar2177 your eyeballs would be appreciated as well

File: collectd-package-install [hunk 1]
```diff
@@ -173,6 +203,7 @@ Enter your Selection: "
 #confirm user input (yes or no)
 confirm ()
 {
+    [ $interactive -eq 0 ] && return
```
Comment (ramjothikumar) [inline collectd-package-install]: When '-y' is not passed in the argument, the prompt for "Is this correct? [y/N]" shows up as expected.
But the prompt for entering the dns or hostname selection does not show up.

This was showing up before this change.

Comment (mdubbyap) [inline collectd-package-install]: how about now?

File: collectd-package-install [hunk 2]
```diff
@@ -3,21 +3,61 @@
...
+              [ -z "$source_type" ] && source_type="-s dns"
+              interactive=0; shift 1 ;;
+           --beta)
+              stage=beta
+              installer_level="-b" ; shift 1 ;;
+           --test)
+              stage=test
+              installer_level="-T" ; shift 1 ;;
+           -H)
+              source_type="-s input -H $2"; shift 2 ;;
```
Comment (ramjothikumar) [inline collectd-package-install]: @mdubbyap Please add some validation for when a user enter -H but leaves the hostname blank.

```[root@2a0886dad312 signalfx-collectd-configs]# ./collectd-package-install rb2yT1_ej4yXYkIdSQzp_w -y -H
 cent6
host




^C```

**Comment by mdubbyap** [comment]
Updated to error out if hostname argument omitted.

**Comment by ramjothikumar** [comment]
Ran through the following combinations which work as expected now.

- ./collectd-package-install token -y -H host1
- ./collectd-package-install token -y -H
- ./collectd-package-install token -y
- ./collectd-package-install token -H host1
- ./collectd-package-install token

But, there is no validation for needing a token. So when a token is not passed, the script does not provide feedback. Corner case.

```[root@f43435478406 signalfx-collectd-configs]# ./collectd-package-install
Install will proceed for CentOS release 6
Is this correct? [y/N] y
Installing Dependencies
Loaded plugins: fastest mirror
```

**Comment by ramjothikumar** [comment]
LGTM.

**Pull request merged**